### PR TITLE
Template multisig security

### PR DESCRIPTION
**Pull request by edouardparis**
20241029_18h22m22s_grim
20241029_18h22m31s_grim

### CODE REVIEW

File: gui/src/installer/view/editor/template/mod.rs [hunk 1]
```diff
@@ -47,6 +48,25 @@ pub fn choose_descriptor_template(progress: (usize, usize)) -> Element<'static,
...
             )
+            .push(
+                card::simple(
+                    Row::new()
+                        .align_items(Alignment::Center)
+                        .push(
+                            Column::new()
+                                .align_items(Alignment::Start)
+                                .push(h3("Multisig security wallet"))
+                                .push(p2_regular("A secure scheme requiring stricter multiparty signature and recovery").style(color::GREY_2))
```
Comment (jp1ac4) [inline gui/src/installer/view/editor/template/mod.rs:59]: nit: The "Simple Inheritance" description above ends with a full stop, but this and the "Custom" description don't.

I would choose without the full stop for now as the descriptions are brief.

**Comment by jp1ac4** [comment]
After setting the two primary keys, when I click on "Set" for the recovery key, I get a crash:

ERROR liana_gui:611: panic occurred at line 235 of file src/installer/step/descriptor/editor/mod.rs: Some("index out of bounds: the len is 2 but the index is 2")

I have one HWW connected.

**Review by jp1ac4 — APPROVED** [review]
Tested ACK https://github.com/wizardsardine/liana/commit/319b7705406114887deeda2d9e618008cf819487.

File: gui/src/installer/view/editor/template/multisig_security_wallet.rs [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+                    .push(p1_regular("Primary key #2").bold())
+                ).push(
+                    Row::new()
+                        .align_items(Alignment::Center)
+                        .spacing(10)
+                        .push(icon::round_key_icon().size(H3_SIZE).style(color::ORANGE))
+                        .push(p1_regular("Recovery key").bold())
+            ))
+            .push(Container::new(
+                p1_regular("The Primary Keys will compose a 2 of 2 multisig which will be your always-active spending policy. In case one of your keys becomes unavailable, after a period of inactivity you will be able to recover your funds using the Recovery Key together with one of your Primary Keys (2 of 3 multisig):")
```
Comment (jp1ac4) [inline gui/src/installer/view/editor/template/multisig_security_wallet.rs:61]: nit: add hyphens, e.g. "2-of-2 multisig"

**Review by edouardparis — COMMENTED** [review]
Self-ACK 36af35fae3f21995fcbe0c4c1a5f685a2318d0a0
rebased on top of Michael ACK

**Pull request merged**